gremlin-go/driver/protocol.go

```diff
@@ -116,19 +116,17 @@ func (protocol *gremlinServerWSProtocol) responseHandler(resultSets *synchronize
 	if statusCode == http.StatusNoContent {
 		resultSets.load(responseIDString).addResult(&Result{make([]interface{}, 0)})
 		resultSets.load(responseIDString).Close()
-		log.logger.Log(Info, "No content.")
+		log.logStr(Info, "No content.")
```

Comment:
```suggestion
```

Reply:
removed in https://github.com/lyndonbauto/tinkerpop/pull/70/commits/379d3af3e9d78076a85faa484750c82463db84c8